slices/slices.go

```diff
@@ -146,10 +149,16 @@ func ToMap[T1 any, T2 comparable](source []T1, lookup func(T1) T2, strictUniquen
 // TryCast tries to cast each element of the slice from type T1 to type T2. Elements get filtered out of the cast is unsuccessful.
```

Comment:
In which use case do you not want to return error if it's not convertible?

Reply:
none, it simply filters out inconvertible values